cms/djangoapps/contentstore/tests/test_course_settings.py

```diff
@@ -1106,3 +1109,66 @@ def test_add(self):
         self.assertEqual(obj, grader)
         current_graders = CourseGradingModel.fetch(self.course.id).graders
         self.assertEqual(len(self.starting_graders) + 1, len(current_graders))
+
+
+class CourseMarketingSiteTest(CourseTestCase):
```

Comment:
This class name and the doc string below are too general. This isn't a base class, and all the test points are specifically about the course end date.